kuma/javascript/src/payments/subscription-form.jsx

```diff
@@ -42,6 +42,42 @@ function useScriptLoading(url) {
     return [loadingPromise, () => setLoadingPromise(null)];
 }
 
+const STRIPE_CONTINUE_SESSIONSTORAGE_KEY = 'stripe-form-continue';
+
+/**
+ * Return true if you have a sessionStorage key that says you can go straight
+ * to continue the Stripe subscription form.
+ * This also has the side-effect that once you call it, it demolishes that
+ * sessionStorage.
+ */
+function getStripeContinuation() {
```

Comment:
nit(ish): Imo `get` is the wrong verb for a destructive read. I don't know what's the best one, but here's some ideas: pop, retrieve, take.
Alternatively I'm also down for explicitness like `getAndUnsetStripeContinuation`

Reply:
Yeah, pop is better. I actually had it like that and then changed my mine. The reason I changed my mind is that I first thought it didn't make sense to use:
```
var value = popSomething()
```
but then I realized, late, that that's how Python works:
```python
>>> list=list('abc')
>>> value = list.pop()
>>> value
'c'
```